stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/preview/MediaGalleryPreviewActivity.kt

```diff
@@ -1559,6 +1598,36 @@ public class MediaGalleryPreviewActivity : AppCompatActivity() {
          */
         private const val KeyVideoThumbnailsEnabled: String = "videoThumbnailsEnabled"
 
+        /**
+         * Represents the key for the [Boolean] value found inside [StreamCdnImageResizing]
+         * turning image resizing on or off.
+         */
+        private const val KeyImageResizingEnabled: String = "imageResizingEnabled"
```

Comment:
The core module isn't an android module and doesn't use `kotlin-parcelize`. 

Instead of moving the models to an android module where they might fight less well, I opted out for a more primitive but equally functional serialization approach.